recipes/libMobility/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "libmobility" %}
+{% set version = "0.5.0" %}
+
+{% set cuda_compiler_version = cuda_compiler_version | default("None") %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/stochasticHydroTools/libMobility/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 44a3298709b8d8ae9ef16b0c49167510a3d49dafe5f4d4dfd6670f365004959c
+
+build:
+  skip: true  # [win or not (cuda_compiler_version or "").startswith("12")]
+  # Debugging skips
+  # skip: true  # [py != 312]
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  rpaths:
+    - lib/
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64]
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - make
+    - cmake >=3.25.2
+    - numpy                                  # [build_platform != target_platform]
+    - nomkl                                  # [build_platform != target_platform]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - numpy
+    - cuda-version {{ cuda_compiler_version }}
+    - cuda-cudart-dev
+    - libcufft-dev
+    - libcublas-dev
+    - libcusolver-dev
+    - libcurand-dev
+    - blas-devel
+    - libblas
+    - liblapacke
+    - nomkl
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+    - scipy
+    # Prove that netlib-linked BLAS works with MKL
+    - libblas=*=*mkl  # [x86 or x86_64]
```

```suggestion
    - libblas * *mkl  # [x86 or x86_64]
```
According to the linter, this is the preferred style